### ISSUE TITLE
fix(whitepaper): pdf rendering in sections 2.1.2, 3, 8

### ISSUE THREAD

**Issue by petermetz** [hyperledger/cactus#339]
**Describe the bug**

Some of the exotic syntax we have in the whitepaper that works fine on github markdown is broken after the make task is used to render the PDF with pandoc. See attached screenshots for examples.

**To Reproduce**

1. Check out the project source code
2. cd whitepaper
3. `make`
4. open the file `hyperledger-cactus-whitepaper.pdf`
5. observe the issues as demonstrated by the screenshots attached as well.

**Expected behavior**

PDF renders more or less the same way as it gets shown on GitHub when viewing the markdown file directly, e.g.:
https://github.com/hyperledger/cactus/blob/master/whitepaper/whitepaper.md

**Logs/Stack traces**

`N/A`

**Screenshots**

On each of these pictures, you can observe the GitHub rendered markdown on the right side of the picture and the PDF with the rendering issues on the left side:

![cactus-whitepaper-pdf-rendering-issue-2020-11-03--01](https://user-images.githubusercontent.com/1583231/98051616-17464a00-1de9-11eb-8a36-4e1fd2280044.png)
![cactus-whitepaper-pdf-rendering-issue-2020-11-03--02](https://user-images.githubusercontent.com/1583231/98051617-17dee080-1de9-11eb-974b-59765908be40.png)
![cactus-whitepaper-pdf-rendering-issue-2020-11-03--03](https://user-images.githubusercontent.com/1583231/98051619-17dee080-1de9-11eb-9319-11da463ca983.png)

**Cloud provider or hardware configuration:**

`N/A`

**Operating system name, version, build:**

`N/A`

**Hyperledger Cactus release version or commit (git rev-parse --short HEAD):**

```sh
$ git rev-parse --short HEAD
1cf5176
```

**Hyperledger Cactus Plugins/Connectors Used**

`N/A`

**Additional context**

The HTML file might also be broken the same way (since that is the one the PDF gets rendered from).

On top of the maintainers, also tagging some past contributors for the whitepaper who might have some information about how to fix these syntax issues, don't feel obligated to contribute, I'm just fishing! =>

cc: @jonathan-m-hamilton @sfuji822 @takeutak @RafaelAPB @hartm @doben @dilumb

**Comment by RafaelAPB** [comment]
Perhaps section's 8 image is too large?

**Comment by petermetz** [comment]
> Perhaps section's 8 image is too large?

@RafaelAPB  Yeah, seems like it. My guess is that it could be fixed by setting an explicit width on it. 
I found out by trial and error for other images that it's around 700 pixels is what pandoc ends up liking for the PDF rendering.

**Comment by AzaharaC** [comment]
For the problem of the tables in point 3, I think the difference is due to how the markdown files look on github. 
If I use the Markdown All in One plugin from Visual Studio Code to see the preview of the file, the table looks like in the pdf:
![image](https://user-images.githubusercontent.com/77972630/116262812-23e4a300-a779-11eb-83cd-518f19da4e2b.png)

**Comment by petermetz** [comment]
@AzaharaC That one is lower priority than the others so as long as the rest is fixed I'd be okay with sweeping the difference in point 3 under the rug. :-)
